content/hardware/build_instructions.md

```diff
@@ -491,7 +485,8 @@ In this section, we will configure the buck converter. The buck converter steps
 
 6. Cut a length of wire of approximately 22 cm. Create this length of wire from the extra wire that was cut off of the barrel connector in step 1. 
 
-7. Using two solderless wire connectors and heat shrink, connect the 22cm length of wire to the battery connector. Use the exact same process that was detailed in the *VESC Preparation* section, particularly steps 2-13.
+7. Using two solderless wire connectors and heat shrink, connect the 22cm length of wire to the battery connector. Use the exact same process that was detailed in the *VESC Preparation* section, particularly steps 2-13. If using the Xavier NX, utilise two battery connectors, and connect the black wire of one of the connectors to the red wire of the other connector, with the other red and black wire connected to the 22cm length of wire. This will allow you to connect two batteries in series in order to provide the higher required voltage for the Xavier NX.
```

Comment:
It would be useful to post a picture here for both the Nano version and the NX version

Reply:
I'll make sure to get these sometime this week and add them.